build/squire-raw.js

```diff
@@ -3576,7 +3576,11 @@ if ( top !== win ) {
         win.onEditorLoad = null;
     }
 } else {
-    win.Squire = Squire;
+    if ( module && module.exports ) {
```

Comment:
did you run this code? I think it will throw `Uncaught ReferenceError: module is not defined`

You need to change it to `if ( win.module && win.module.exports ) {`